Examples/Widgets/Notes/src/notesWidget.ts

```diff
@@ -0,0 +1,344 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  PropertyValues,
+  query,
+  TemplateResult,
+} from "lit-element";
+import { nothing } from "lit-html";
+import { classMap } from "lit-html/directives/class-map";
+import app from "./assets/localisation/app.json";
+import "./components";
+import {
+  INITIAL_FILTER,
+  NotesFilter,
+} from "./components/NotesWidgetHeader/NotesWidgetHeader";
+import {
+  COLORS,
+  DATE,
+  DELETE,
+  EMPTY_STATE_ILLUSTRATION,
+  FAVORITES,
+  MAX_NOTES_COUNT,
+} from "./constants";
+import style from "./assets/styles/NotesWidget.scss";
+import { fetchNotes, getBlankNote, getTimestampInDateLocale } from "./Utils";
+
+export let agentId = "";
+
+export namespace NotesWidget {
+  /**
+   * @element notes-widget
+   */
+  @customElement("notes-widget")
+  export class MyCustomComponent extends LitElement {
+    @property({ type: String, attribute: "agent-id" }) agentId = "";
+
+    @internalProperty() private isFeatureFlagEnabled = false;
+    @internalProperty() private showErrorState = false;
+    @internalProperty() private originalNotes: Note[] = [];
+    @internalProperty() private notes: Note[] = [];
+    @internalProperty() private selectedNoteDetails:
+      | Note
+      | undefined = undefined;
+    @internalProperty() private isEditMode = false;
+    @internalProperty() private selectedNotes: Record<string, boolean> = {};
+    @internalProperty() private action = "";
+    @internalProperty() private filter: NotesFilter = { ...INITIAL_FILTER };
+
+    @query(".notes-container") private notesContainer!: any;
+
+    protected updated(changedProperties: PropertyValues): void {
+      super.updated(changedProperties);
+      if (changedProperties.has("agentId")) {
+        this.updateAgentId();
+      }
+    }
+
+    protected firstUpdated(changedProperties: PropertyValues): void {
+      super.firstUpdated(changedProperties);
+      // SERVICE.telemetry.track(SERVICE.telemetry.MIX_EVENT.NOTES_WIDGET_OPENED);
```

Comment:
Dont leave thsi in comments...remove it completely

Reply:
> Dont leave thsi in comments...remove it completely

Changes Updated.